services/report/raw_upload_processor.py

```diff
@@ -153,9 +153,13 @@ def process_raw_upload(
                 path_fixer_to_use,
                 should_use_encoded_labels,
             )
-            report = process_report(
-                report=report_file, report_builder=report_builder_to_use
-            )
+            try:
+                report = process_report(
+                    report=report_file, report_builder=report_builder_to_use
+                )
+            except ReportExpiredException as r:
+                r.filename = current_filename
```

Comment:
Test uncovered in case we want to make a test for this

Reply:
fixed!